### DISCUSSION TITLE
Difficulty generating the docker compose.yml file.

### DISCUSSION THREAD

**Discussion by imamahmed28** [tech-acs/chimera-docs#17] [Q&A] [answered]
Hello, 
Whenever, i run the "php artisan chimera: dockerize", I get "could not open input file: artisan"
I am installing it using docker

**Comment by baakal** [comment, marked as answer]
The php artisan chimera: dockerize, command is used to generate the dockerfile, docker-compose.yml file and runtime config files. To use this command you need to install php, composer. you will have to also initialize the dashboard using the starter-kit, follow the [documentation](https://tech-acs.github.io/chimera-docs/docs/developer/getting-started/installation) and the following environment setup is required for development [link](https://tech-acs.github.io/chimera-docs/docs/developer/getting-started/environment-setup).

Once you are ready for deployment you can use php artisan chimera:dockerize to generate the necessary files for deployment (in a test or production docker environment).

The misleading documentation has been corrected to reflect the correct information on this PR https://github.com/tech-acs/chimera-docs/pull/18
